src/GoProControl.cpp

```diff
@@ -495,6 +513,30 @@ bool GoProControl::isConnected(const bool silent)
   }
 }
 
+  //ADD DAMIEN
+void GoProControl::getConnection()
```

Comment:
what is the difference with isConnected?

Reply:
isConnected can read the wifi status only if a command as been sent. I nothing happen and the camera disconnect, we will have to wait another command to see that the camera is no longer connected.
getConnection test if the wifi is still "here", and update the variable _connected. I can remove from here if you want to, I will keep it in my repo ^^